myuw/templates/handlebars/card/seattle_application.html

```diff
@@ -0,0 +1,116 @@
+			<div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Email</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content"><a href="mailto:regoff@uw.edu">regoff@uw.edu</a></div>
+                </div>
+            </div> 
+			<div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Phone</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">(206) 543-4000</div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Fax</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">(206) 616-3222</div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Location</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">225 Schmitz Hall</div>
+                </div>
+            </div>
+            <div class="row last">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Mail</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">Registration and Transcripts Office<br>University of Washington <br>Box 355850 <br>Seattle, WA 98195-5850</div>
+                </div>
+            </div>			
+	</div>
+	<div class="myuw-card-section">	
+
+		<h4 class="applicant-card"> Resources for Seattle Applicants</h4>
+			<ul class="unstyled-list list-leading">
+			    <li><a href="https://www.washington.edu/financialaid/" target="_blank">Learn all about student financial aid and scholarships</a></li>
+			    <li><a href="http://www.washington.edu/students/" target="_blank">Check out the Student guide for academics, student life, and more</a></li>
+			   	<li><a href="http://www.washington.edu/students/reg/calendar.html" target="_blank">View the UW Academic calendars</a></li>
+			</ul>
+
+
+    </div>
+
+{{else}}   
+    <h3 class="myuw-card-title">
+        Your Seattle Application for {{quarter}} {{year}}
+    </h3>
+
+    <div class="myuw-card-section">
+        
+        <div class="clearfix">
+             <span class="btn btn-default card-action-btn myuw-btn-big"><a href="https://sdb.admin.uw.edu/admissions/uwnetid/appstatus.asp"  data-linklabel="Your application status details"><h4 class="applicant-card">View your {{type}} application status</h4></a></span>
+		</div>
+		<div class="links-container actionlink-list">
+			<h4 class="applicant-card"> Resources for Seattle Applicants</h4>
```

Comment:
Couple thoughts on this:

1. _Implementation_: I think it would be better to use or create a more generalizable heading style here. Redefining a heading for this specific card means it can't be reused anywhere else. Something like `myuw-card-heading` perhaps (which is defined already for the `.card` cards btw, we could just port it over for `.myuw-card` too.

2. _Needed?_: Are we sure this heading is even needed? The subheadings provide more descriptive categories and the fact that they are lists of links makes it seem clear it is resources.

3. _Style_: If you decide to keep the heading, I think it is worth exploring the other heading styles we are using elsewhere. That Roboto style heading seems like it is only used as the sub-heading text for course names and on the Quick Links card. On the Account page cards we just made we're using these heading styles above lists of links/resources.
![myuw__accounts](https://user-images.githubusercontent.com/6953502/32809719-d352f3a0-c94c-11e7-8a98-582005053130.jpg)

Reply:
RE: 2. Needed?: I think so. Applicants are very new to UW and college stuff, thus contextualizing more would be beneficial. 
3. Style -- Agreed with the general approach of using the same sub-heading style everywhere. For the consistent heading style on the home page, it was matched to the Quicklinks card. We can change both sub-headings.